list/LIST_CONSULTATION_README.md

```diff
@@ -0,0 +1,209 @@
+# List (Consultation) Mapping
+
+## XML HL7 > JSON FHIR
+
+A List (Consultation) is created from mapped [Encounters](../encounters/README.md). Where the EHR extract is used it is the root `EhrExtract` of the XML.
+This is a top-level profile and represents the whole structured consultation
+
+| Mapped to (JSON FHIR List field) | Mapped from (XML HL7 / other source)                                                  |
+|----------------------------------|---------------------------------------------------------------------------------------|
+| id                               | `Encounter / id [@root]`                                                              |
+| meta.profile\[0]                 | fixed value = `"https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-List-1"` |
+| status                           | fixed value = `current`                                                               |
+| mode                             | fixed value = `snapshot`                                                              |
+| title                            | `Encounter / type`                                                                    |
```

Comment:
`Encounter / type\[0] / text` or `Encounter / type\[0] / coding\[0] / display`

Reply:
fixed as now references as to how these fields are derived from the XML in the encounter mapping